docs/working-group/notes/notes_12-4-2020.md

```diff
@@ -0,0 +1,29 @@
+# Karpenter working group meeting (12/4/2020)
```

Comment:
Do you mind adding this to the working group README instead of a notes folder? It tends to be easier to scroll/search if the notes are in one doc.

Reply:
Won't it get super long in few weeks/months?

Reply:
Yeah -- maybe we can do per quarter or per year? It turns out that weekly isn't really that much text per year. Especially if we move to bi-weekly. This just comes from my experience w/ running a WG Meeting doc from KFServing.